npm.json

```diff
@@ -9,5 +9,6 @@
         }
       ]
     }
-  ]
+  ],
+  "postUpdateOptions": ["yarnDedupeHighest", "npmDedupe", "pnpmDedupe"]
```

Comment:
I take it we tested with the full array and it works for all package managers? Exciting!